seismic_hazard_analysis/nshm_2022/oq_interface.py

```diff
@@ -0,0 +1,236 @@
+import multiprocessing as mp
+import time
+from collections.abc import Sequence
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import xarray as xr
+from openquake.calculators.extract import Extractor
+from openquake.commonlib.datastore import read
```

Comment:
Shadowing the builtin `read` here is probably not a good idea.


```suggestion
from openquake.commonlib import datastore
```